tests/modules/services/window-managers/bspwm/bspwmrc

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
we shouldnt have these empty lines.

Reply:
Fixed in 1d2a8100cf460c2a3990a37ede135292c7b19e80, the if statements are there to make sure there's a newline in between `extraConfigEarly` and the rest.